spec/lib/search/query_builder_spec.rb

```diff
@@ -446,8 +446,8 @@
       context "if no custom serving config is given" do
         let(:v2_serving_config) { nil }
 
-        it "includes the default serving config query" do
-          expect(query).to include("serving_config" => "default_search")
+        it "does not include a serving config query" do
+          expect(query).not_to have_key("serving_config")
```

Comment:
Shouldn't this be `include("serving_config" => nil)` rather than the key not existing at all?

Reply:
No, as we `#compact` the params in `QueryBuilder#v2_serving_config_query` so it won't get sent through at all 🚫

Reply:
Ah! I missed that. Thanks!